app/models/early_career_payments/eligibility.rb

```diff
@@ -1,5 +1,128 @@
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2019),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 5_000,
+        uplift_amount: 7_500
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :physics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :chemistry,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :foreign_languages,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2022),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2018),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 5_000,
+        uplift_amount: 7_500
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :physics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :chemistry,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :foreign_languages,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2023),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2019),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 5_000,
+        uplift_amount: 7_500
+      ),
+      AwardAmount.new(
+        itt_subject: :mathematics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
+        uplift_amount: 3_000
+      ),
+      AwardAmount.new(
+        itt_subject: :physics,
+        itt_academic_year: AcademicYear.new(2020),
+        claim_academic_year: AcademicYear.new(2024),
+        base_amount: 2_000,
```

Comment:
@nomasprime Any reason for not extracting this into its own class? Feels like its a smell sitting in the policy::Eligibility class now.